javascript/javascript2/week2/movies.js

```diff
@@ -0,0 +1,320 @@
+  },
+  {
+    title: "11-11-11",
+    year: 2011,
+    rating: 4,
+    votes: 11425,
+    running_times: 5400,
+  },
+  {
+    title: "11:14",
+    year: 2003,
+    rating: 7.2,
+    votes: 40149,
+    running_times: 5700,
+  },
+  {
+    title: "12 Angry Men",
+    year: 1957,
+    rating: 8.9,
+    votes: 518449,
+    running_times: 5760,
+  },
+  {
+    title: "12 Rounds",
+    year: 2009,
+    rating: 5.6,
+    votes: 24457,
+    running_times: 6480,
+  },
+  {
+    title: "12 Years a Slave",
+    year: 2013,
+    rating: 8.1,
+    votes: 513047,
+    running_times: 8040,
+  },
+  {
+    title: "127 Hours",
+    year: 2010,
+    rating: 7.6,
+    votes: 302809,
+    running_times: 5640,
+  },
+  { title: "13", year: 2010, rating: 6.1, votes: 36623, running_times: 5460 },
+  {
+    title: "13 Going on 30",
+    year: 2004,
+    rating: 6.1,
+    votes: 134434,
+    running_times: 5880,
+  },
+  {
+    title: "13 Hours",
+    year: 2016,
+    rating: 7.3,
+    votes: 85771,
+    running_times: 8640,
+  },
+  {
+    title: "13 Sins",
+    year: 2014,
+    rating: 6.3,
+    votes: 26738,
+    running_times: 5580,
+  },
+  { title: "13th", year: 2016, rating: 8.2, votes: 15188, running_times: 6000 },
+  {
+    title: "1408",
+    year: 2007,
+    rating: 6.8,
+    votes: 228738,
+    running_times: 6840,
+  },
+  {
+    title: "1492: Conquest of Paradise",
+    year: 1992,
+    rating: 6.5,
+    votes: 24439,
+    running_times: 9240,
+  },
+  {
+    title: "15 Minutes",
+    year: 2001,
+    rating: 6.1,
+    votes: 44647,
+    running_times: 7200,
+  },
+  {
+    title: "16 Blocks",
+    year: 2006,
+    rating: 6.6,
+    votes: 117443,
+    running_times: 6600,
+  },
+  {
+    title: "17 Again",
+    year: 2009,
+    rating: 6.4,
+    votes: 156663,
+    running_times: 6120,
+  },
+  { title: "1922", year: 2017, rating: 6.4, votes: 15416, running_times: 6120 },
+  { title: "1941", year: 1979, rating: 5.9, votes: 27172, running_times: 8760 },
+  {
+    title: "2 Days in New York",
+    year: 2012,
+    rating: 6,
+    votes: 12766,
+    running_times: 5760,
+  },
+  {
+    title: "2 Days in Paris",
+    year: 2007,
+    rating: 6.8,
+    votes: 28106,
+    running_times: 5760,
+  },
+  {
+    title: "2 Days in the Valley",
+    year: 1996,
+    rating: 6.5,
+    votes: 16528,
+    running_times: 6240,
+  },
+  {
+    title: "2 Fast 2 Furious",
+    year: 2003,
+    rating: 5.9,
+    votes: 213674,
+    running_times: 6420,
+  },
+];
+// filtering as short title 
+const shortTitleMovies = movies.filter(item => item.title.length <= 5);
+// console.log(shortTitleMovies);
+
+
+// filtering as long title 
+const longTitleMovies = movies.filter((item) => item.title.length >= 10);
+// console.log(longTitleMovies);
+
+
+// filtering as year  
+const counter = movies.filter( item => item.year >= 1980 && item.year <= 1989 
+
+)
+console.log(counter.length);
+
+// adding tag prop to objects 
+const tagArray = movies.forEach(element => {
```

Comment:
Think of using `filter` method
```suggestion
const tagArray = movies.filter(element => {
```